crates/subspace-node/src/genesis_allocations.rs

```diff
@@ -0,0 +1,35 @@
+use subspace_runtime_primitives::{AccountId, Balance};
+use sp_core::crypto::Ss58Codec;
+use serde_json::Value;
+use sp_runtime::traits::Zero;
+
+pub fn get_genesis_allocations() -> Vec<(AccountId, Balance)> {
+    let file_content = include_str!("genesis_allocations.json");
```

Comment:
We typically store such things in a constant at the top of the file, not in local variable, see the chain spec example. Also I'd make this function take contents as an argument with constant that contains JSON being in the chain spec file.